src/scss/tooltip.scss

```diff
@@ -1,4 +1,7 @@
 @import "o-overlay/main";
+//
+$o-typography-is-silent: true;
```

Comment:
Components should be  silent by default, clients silent mode off.
https://github.com/Financial-Times/o-typography/blob/5295a719c285ee1b22101795d62df079ca0f40d3/src/scss/_variables.scss#L8